e2e/utils/tenv.ts

```diff
@@ -301,7 +331,9 @@ export function createTenv(cwd: string): Tenv {
       ]);
       return service;
     },
-    async container({ name, image, env = {}, containerPort, healthcheck, pipeLogs }) {
+    async container({ name, image, env = {}, containerPort, healthcheck, pipeLogs, volumes = [] }) {
+      checkLocalDockerImage(image);
```

Comment:
What is this check for?

Reply:
mainly for debugging, it logs in the console whether the image used was found locally or not. that's it